### ISSUE TITLE
#-1550789047------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#1563]
<strong>Source:</strong>`<a id="edit-button-e97cdf5982344149b666d0832d4ce521" href="/layers/e97cdf5982344149b666d0832d4ce521/edit" aria-describedby="calcite-tooltip-749d60d0-2805-25aa-a7be-594e4a1cd4e1">`

<strong>Element Location: </strong>#edit-button-e97cdf5982344149b666d0832d4ce521

<strong>Impact: </strong>serious

<strong>Failure Summary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

<strong>Tags:</strong> cat.name-role-value,wcag2a,wcag412,wcag244,section508,section508.22.a,ACT

<strong>HelpUrl:<strong>https://dequeuniversity.com/rules/axe/4.6/link-name?application=axeAPI
